ouroboros-network-framework/sim-tests/Test/Ouroboros/Network/Server/Sim.hs

```diff
@@ -226,7 +226,7 @@ data ConnectionEvent req peerAddr
     -- ^ Close an outbound connection.
   | ShutdownClientServer DiffTime peerAddr
     -- ^ Shuts down a client/server (simulates power loss)
-  deriving (Show, Functor)
+  deriving (Eq, Show, Functor)
```

Comment:
apropos `Improve Simulation testnet shrinker`: could you explain where the clue of the fix is?